.github/workflows/metadata_diff.yml

```diff
@@ -89,42 +103,72 @@ jobs:
+          chmod +x $HOME/polkadot
           ls -ahl $HOME/
-      - name: Start old node
+      - name: stop sccache server
+        run: sccache --stop-server || true
+      - name: Install pm2
+        run: npm install -g pm2
+      - uses: actions/checkout@v2
+        with:
+          repository: paritytech/polkadot-launch
+          path: polkadot-launch
+      - name: create launch config
         run: |
-          echo Running on $CHAIN
-          $HOME/manta-old --chain=$CHAIN --port 30444 --base-path ./temp1 --ws-port 9944 -- --chain kusama &
-      - name: Start new node
+          cd ${{ github.workspace }}/polkadot-launch
+          cat ${{ github.workspace }}/.github/resources/config-for-integration-test.json | \
+            jq \
+              --arg relaychain_bin $HOME/polkadot \
+              --arg parachains_bin $HOME/manta \
+              --arg parachains_spec $CHAIN \
+              '.relaychain.bin = $relaychain_bin
+              | .parachains[0].bin = $parachains_bin
+              | .parachains[].chain = $parachains_spec
+              ' > ${{ github.workspace }}/.github/resources/diff-metadata.json
+      - name: launch testnet
         run: |
-          echo Running on $CHAIN
-          target/release/manta --chain=$CHAIN --port 30555 --base-path ./temp2 --ws-port 9955 -- --chain kusama &
+          cd ${{ github.workspace }}/polkadot-launch
+          yarn install
+          yarn build
+          pm2 start dist/cli.js --name polkadot-launch \
+            --output ${{ github.workspace }}/stdout.log \
+            --error ${{ github.workspace }}/stderr.log \
+            -- ${{ github.workspace }}/.github/resources/diff-metadata.json
       - name: Prepare output
         run: |
-          VERSION=$(./target/release/manta --version)
+          VERSION=$($HOME/manta --version)
           echo "Metadata comparison:" >> output.txt
           echo "Date: $(date)" >> output.txt
           echo "Target version: $VERSION" >> output.txt
           echo "Chain: $CHAIN" >> output.txt
           echo "----------------------------------------------------------------------" >> output.txt
-      - name: Pull polkadot-js-tools image
-        run: docker pull wilwade/polkadot-js-tools
-
+      - uses: actions/checkout@v2
+        with:
+          repository: polkadot-js/tools
+          path: tools
       - name: Compare the metadata
         run: |
-          CMD="docker run --pull always --network host wilwade/polkadot-js-tools metadata ws://localhost:9944 ws://localhost:9955"
+          cat ${{ github.workspace }}/stdout.log
+          cat ${{ github.workspace }}/stderr.log
+          ENDPOINT=wss://calamari.systems
+          if [[ "$CHAIN" == *"manta"* ]]; then
+            ENDPOINT=wss://ws.manta.systems
+          fi;
+          echo $ENDPOINT
+          cd ${{ github.workspace }}/tools
+          yarn
+          CMD="yarn run:metadata $ENDPOINT ws://127.0.0.1:9921"
```

Comment:
i can't remember if there was a good reason for comparing to a local node previously ... instead of to the production node